packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -588,6 +589,13 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     ev.preventDefault();
   };
 
+  private _onKeyDown = (ev: React.KeyboardEvent<HTMLDivElement | HTMLAnchorElement | HTMLButtonElement>) => {
+    if (this.props.disabled && (ev.which === KeyCodes.enter || ev.which === KeyCodes.space)) {
+      ev.preventDefault();
+      ev.stopPropagation();
+    }
+  };
+
```

Comment:
Can this be unit tested? (not sure if synthetic key events fall back to calling click)

Reply:
Added unit test in button.test.tsx but there's a problem - commented out the last line of the test for now expect(didClick).toEqual(false); because it's finding true even though I'm unable to click a disabled button and repro it on the page itself...